ir/src/test/scala/org/scalajs/core/ir/PrintersTest.scala

```diff
@@ -975,11 +975,40 @@ class PrintersTest {
 
   @Test def printPropertyDef(): Unit = {
     assertPrintEquals(
-        """<property: "prop">""",
+        """
+          |get "prop"(): int = {
```

Comment:
Getters are supposed to always return `any`, since they are called from JS. The displayed result type should be `any`.